swagger.yaml

```diff
@@ -5021,6 +5021,9 @@ definitions:
         x-nullable: true
       load_balancer:
         $ref: '#/definitions/load_balancer'
+      consumer:
```

Comment:
Is it necessary to change the API? I think if we can avoid it it would be best, as I can't really think of any other scenario where this is needed.

Reply:
Hmm I guess we can avoid it by either:

- adding another optional parameter during create/update cluster 
- or creating a new function to just store this value 

What do you think?

Reply:
Why do we need to add a new parameter? Can we use the tags field for this?
https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L5017 
the UI is using it and we have it in our data